css/style.css

```diff
@@ -1039,22 +1041,1370 @@ li { list-style-type: none; }
+    margin: 0 34.51px 0 0;
+    text-transform: uppercase;
+  }
+
+  .test-drive__link:hover {
+    display: flex;
+    font-size: 16px;
+    line-height: 20px;
+    font-weight: bold;
+    text-align: center;
+    text-decoration: none;
+    color: #000;
+    cursor: pointer;
+    padding: 0;
+    border: 0;
+  }
+
+  .test-drive__button {
+    display: flex;
+    width: 221px;
+    height: 60px;
+    align-items: center;
+    justify-content: center;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 16px;
+    line-height: 20px;
+    text-align: center;
+    border: 2px solid #fff;
+    color: #fff;
+    background-color: #f00;
+  }
+
+  .test-drive__button:hover {
+    font-size: 14px;
+    color: #fff;
+    border: 3px solid #fff;
+    background-color: #f00;
+    cursor: pointer;
+    font-weight: bold;
+    padding: 0;
+  }
+
+  .test-drive__button:focus {
+    display: flex;
+    color: #fff;
+    border: 1px solid #fff;
+    background-color: #f00;
+    cursor: pointer;
+    font-weight: bold;
+    padding: 0;
+    outline: none;
+    box-shadow: 0 0 0 3px;
+  }
+
+  .test-drive__button:active {
+    background-color: #fff;
+    border-color: #000;
+    color: #000;
+    font-size: 12px;
+  }
+
+  .section__btn-pointer {
+    display: flex;
+    width: 700px;
+    height: 20px;
+    padding: 0;
+    margin: 0 auto 61px auto;
+    box-shadow: none;
+    background: url(https://i.ibb.co/GFX8Xs7/Vector-1.png) no-repeat;
+    background-size: 5%;
+    background-position: center center;
+    outline: none;
+  }
+
+  /* -------2-------- */
+
+  .sections__dashboard {
+    display: flex;
+    flex-flow: column wrap;
+    max-width: 768px;
+    height: 988.03px;
+    align-content: flex-end;
+    margin: 0;
+    background: url(https://i.ibb.co/JkTYWY9/2013-tesla-model-s-cockpit-i-pad.png) no-repeat;
+    background-size: 100.5% 51%;
+    background-position: top left;
+  }
+
+  .dashboard__header {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    max-width: 375px;
+    max-height: 123px;
+    font-size: 70px;
+    line-height: 85px;
+    font-weight: 600;
+    color: #f00;
+    margin: 7.3px 0 0 390px;
+  }
+
+  .dashboard-right {
+    display: flex;
+    flex-flow: row wrap;
+    max-width: 768px;
+    max-height: 487.21px;
+    align-items: center;
+    justify-content: center;
+    font-size: 40px;
+    color: white;
+  }
+
+  .dashboard-right__sidebar {
+    display: flex;
+    flex-flow: row wrap;
+    align-content: flex-start;
+    justify-content: flex-start;
+  }
+
+  .sidebar {
+    display: flex;
+    flex-flow: row wrap;
+    align-content: flex-start;
+    justify-content: flex-start;
+    width: 768px;
+    max-height: 487.21px;
+    color: #fff;
+    background: #f00;
+    background-size: 100%;
+  }
+
+  .sidebar__title {
+    display: flex;
+    justify-content: center;
+    width: 316px;
+    height: 37px;
+    color: #fff;
+    margin: 55.58px 72px 20.51px 34px;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Hi-Tesla/pullRequest?prid=5851407)